nixos/modules/services/monitoring/prometheus/exporters/pihole.nix

```diff
@@ -7,6 +7,13 @@ let
 in
 {
   port = 9617;
+  imports = [
+    # removed option 'services.prometheus.exporters.pihole.interval'
+    (mkRemovedOptionModule [ "interval" ] ''
+      As of prometheus-pihole-exporter 0.3.0 this option no-longer exists.
+    '')
+     ({ options.warnings = options.warnings; options.assertions = options.assertions; })
```

Comment:
Why do we need this?

Reply:
If you mean the `({ options.wa ...` it's because otherwise the docs refuse to build. I suspect this is a deficet in the way the modules are done for the exporters, which I feel are out-of-scope to fix in this PR. I only learned it was necessary by  looking at other uses of mkRemovedOptionModule in the other prometheus exporters.